### DISCUSSION TITLE
Fix this build one device issue

### DISCUSSION THREAD

**Discussion by AbrarNoob** [ROM-builders/temporary#9714] [General]
https://cirrus-ci.com/task/5840024498339840

**Comment by matheucomth** [comment]
Yes for me too

**Comment by matheucomth** [comment]
https://cirrus-ci.com/build/5892157818011648

**Comment by matheucomth** [comment]
@Apon77 sir this is error or feature?

**Comment by Apon77** [comment]
> https://cirrus-ci.com/task/5840024498339840

use lunch or brunch command only once
